docs/output.md

```diff
@@ -622,6 +623,22 @@ Generally you will want to refer to the `combined_reports.txt` or `_report.tre`
 
 You will only receive the `.all`, `.lca`, and `.unc` files if you supply the `--ganon_save_readclassifications` parameter to the pipeline.
 
+### sylph
+
+[sylph](https://github.com/bluenote-1577/sylph) is a program that performs metagenomic profiling or containment average nucleotide identity (ANI) querying for metagenomic shotgun sequencing samples.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `sylph/`
+  - `<db_name>/`
+    - `<sample_id>.sylph.tsv`: output of `sylph profile` containing taxonomic classifications where each row is one genome detected in the sample. This output does not have any information about the species, genus, order etc
+    - `<sample_id>.sylphmpa`: output of `sylph-tax taxprof containing a taxonomic profile like Kraken or MetaPhlan.
+
+</details>
+
+For further descriptions of the contents of each file, see the [sylph documentation](https://sylph-docs.github.io/Output-format/).
```

Comment:
Missing reference to the combined reports from `SYLPHTAX_MERGE` (e.g. with GANON's `_combined_reports.txt` file - it should be under `slyph` alongside `<db_name>`)

Reply:
Added description.